src/components/dialog/createDialogElement.ts

```diff
@@ -0,0 +1,47 @@
+import { CustomElementDescriptor } from '@dojo/widget-core/customElements';
+import createDialog from './createDialog';
+
+/**
+ * Configures a Dialog web component
+ *
+ * @return	{CustomElementDescriptor?}
+ */
+export default function createDialogElement(): CustomElementDescriptor {
+	return {
+		tagName: 'dojo-dialog',
+		widgetFactory: createDialog,
+		attributes: [
+			{
+				attributeName: 'enterAnimation'
+			},
+			{
+				attributeName: 'exitAnimation'
+			},
+			{
+				attributeName: 'modal',
+				value: value => Boolean(value)
```

Comment:
These Boolean conversions aren't working quite right. `Boolean('false')` and `Boolean('0')` return `true`.